keyboards/sets3n/kk980/readme.md

```diff
@@ -0,0 +1,19 @@
+# kk980
+
+![kk980](https://imgur.com/a/9JJjC77)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [sets3n](https://github.com/sets3n)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```

Comment:
These should be filled out.

Also, could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Thank you, I learned a lot.

Reply:
This is very helpful for me.